viya_ark_library/k8s/sas_kubectl.py

```diff
@@ -82,13 +82,7 @@ def __init__(self, executable: Text = "kubectl", namespace: Text = None, global_
                 self.ingress_ns = None
 
                 # get ingress namespace
-                if ingress_namespace is not None:
-                    ns: AnyStr = self.do("get namespace " + ingress_namespace, ignore_errors=True, warning=False)
-                    if len(ns) > 0:
-                        raise NamespaceNotFoundError(
-                            f"The ingress namespace [{ingress_namespace}] was not found in the target environment.")
-                    self.ingress_ns = ingress_namespace
-                else:
+                if ingress_namespace is None:
```

Comment:
`if not ingress_namespace:` might be safer  in the event it is empty string ""

Reply:
Code updated.